src/Native/gc/sample/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 include_directories(..)
 include_directories(../env)
 
+add_definitions(-DNO_STRESS_LOG)
```

Comment:
Hmm, it must have been some temporary issue when I was making the change. It is not needed anymore.